### PR TITLE
avoid null exception if there is empty content on the placement file

### LINKED ISSUE #12071: exception if there is empty content on the placement file
### Describe the bug

Read the title

### PR DESCRIPTION
**Pull request by infofromca**
… content on the placement file
fix #12071

### CODE REVIEW

File: src/OrchardCore/OrchardCore.DisplayManagement/Descriptors/ShapePlacementStrategy/ShapePlacementParsingStrategy.cs [hunk 1]
```diff
@@ -70,6 +70,8 @@ private void ProcessFeatureDescriptor(ShapeTableBuilder builder, IFeatureInfo fe
 
         private void ProcessPlacementFile(ShapeTableBuilder builder, IFeatureInfo featureDescriptor, PlacementFile placementFile)
         {
+            if (placementFile == null) return;
```
Comment (CrestApps) [inline src/OrchardCore/OrchardCore.DisplayManagement/Descriptors/ShapePlacementStrategy/ShapePlacementParsingStrategy.cs:73]: I think you should figure out why `placementFile` is null instead. Check out why `serializer.Deserialize<PlacementFile>(jtr);` is returning null.
You may want to add curly brackets. I think that is becoming the new standard in OC.

Comment (hishamco) [reply src/OrchardCore/OrchardCore.DisplayManagement/Descriptors/ShapePlacementStrategy/ShapePlacementParsingStrategy.cs:73]: I dislike checking the params in `private` functions. As @CrestApps said, try to figure what leading the placement file to be `null`

Comment (ns8482e) [reply src/OrchardCore/OrchardCore.DisplayManagement/Descriptors/ShapePlacementStrategy/ShapePlacementParsingStrategy.cs:73]: null should be checked line # 63 not here

**Review by ns8482e — APPROVED** [review]

**Review by hishamco — APPROVED** [review]

**Comment by hishamco** [comment]
The credit for this PR should go to Seb ;)

**Comment by ns8482e** [comment]
don't merge label can be removed

**Pull request merged**